parsers/gob.go

```diff
@@ -0,0 +1,43 @@
+package parsers
+
+import (
+	"encoding/gob"
+	"fmt"
+	"os"
+
+	"github.com/ablqk/littre-bot/src/dictionary"
+)
+
+var (
+	gobFile = "bin/dict.gob"
+)
+
+func ParseGob() ([]dictionary.Entry, error) {
+	f, err := os.Open(gobFile)
+	if err != nil {
+		return nil, fmt.Errorf("unable to open gob: %w", err)
+	}
+	defer f.Close()
+	decoder := gob.NewDecoder(f)
+	dict := make([]dictionary.Entry, 0)
+	err = decoder.Decode(&dict)
```

Comment:
Does this line work if your slice is not initialised?

Reply:
It _seems_ so: https://golang.org/pkg/encoding/gob/#example__encodeDecode

Reply:
Then l.22 should just be a var declaration.